libs/amplify/amplify/#current-cloud-backend/api/reversehand/build/resolvers/ArchivedAdvert.bids.req.vtl

```diff
@@ -0,0 +1,68 @@
+#if( $ctx.source.deniedField )
```

Comment:
Is there not a better place for this to exist?

Also this all looks commented out?